lib/key.rb

```diff
@@ -1,25 +1,25 @@
 class Key
-  attr_reader :number, :keys
```

Comment:
Don't need this in attr_reader since I can just put it as part of my new hash method